python-checks/src/test/resources/checks/robustCipherAlgorithm.py

```diff
@@ -81,20 +82,52 @@ def pydesExamples():
+  ctx.set_ciphers(ciphers)  # Noncompliant
+# ^^^^^^^^^^^^^^^
+  ciphers2 = ciphers
+  ctx.set_ciphers(ciphers2)  # Noncompliant
+  ciphers3 = 'ECDHE-RSA-AES256-SHA'
+  ctx.set_ciphers(ciphers3)  # Noncompliant
+  ciphers4 = 'ECDHE:RSA:AES256:SHA'
+  ctx.set_ciphers(ciphers4)  # Noncompliant
+  ciphers5 = 'ECDHE:RSA:AES256:SHA:ECDHE-RSA-AES256-SHA'
+  ctx.set_ciphers(ciphers5)  # Noncompliant
+
+def pycryptodome_compliant():
   from Crypto.Cipher import AES
   key = b'Sixteen byte key'
   cipher = AES.new(key, AES.MODE_CCM) # Compliant
 
-def pycaCompliant():
+def pyca_compliant():
   import os
   from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
   from cryptography.hazmat.backends import default_backend
   key = os.urandom(16)
   iv = os.urandom(16)
   aes2 = Cipher(algorithms.AES(key), modes.CBC(iv), backend=default_backend()) # Compliant
 
-def pycryptoCompliant():
+def pycrypto_compliant():
   from Crypto.Cipher import *
   aes1 = AES.new('This is a key123', AES.MODE_CBC, 'This is an IV456') # Compliant
 
+def pyssl_compliant():
+  import ssl
+  ciphers = 'ECDHE-RSA-AES256-GCM-SHA384'
+  ctx = ssl.SSLContext(ssl.PROTOCOL_SSLv23)
+  ctx.set_ciphers(ciphers)
+  ciphers2 = ''
+  ctx.set_ciphers(ciphers2)
+  ciphers3 = 'ECDHE:RSA:AES256:GCM:SHA384'
+  ctx.set_ciphers(ciphers3)
+
+  ciphers4 = 1
+  ctx.set_ciphers(ciphers4)
+  ctx.set_ciphers(null_value)
```

Comment:
So here it is not valid Python code. null_value is not defined. Either you want to do the test with `None` or you want to test the fact that passing a value you don't know should not raise an issue. In this case you should have:
```python
def pyssl_compliant(unknown_cipher):
    ...
    ctx.set_ciphers(unknown_cipher)
```